doc/tutorials/using_agl_as_uos.rst

```diff
@@ -0,0 +1,120 @@
+.. _using_agl_as_uos:
+
+Using AGL as the User OS
+########################
+
+This tutorial describes the steps to run Automotive Grade Linux (AGL) 
+as the User OS on ACRN hypervisor and the problems we got at current stage. 
```

Comment:
This is clearly a work in progress document that does not have a happy ending (yet). Will the issues mentioned here be resolved any time soon?  I would think a tutorial document would show how to do something successfully.  @hwang37 Let me know if I should edit this for publishing in its current state.

Reply:
I'm fine to publish it now without a happy ending.
The reason is that the bug is in AGL side and we need AGL community to patch and fix in AGL repo, out of ACRN project's scope.
Shouldn't we leave some improvement room to AGL community? :)